README.md

```diff
@@ -3,23 +3,24 @@ The Container Storage Interface (CSI) Driver for IBM block storage systems enabl
 
 This is the official operator to deploy and manage IBM block storage CSI driver.
 
-Supported container platforms:
-  - OpenShift v4.4
-  - OpenShift v4.5
-  - Kubernetes v1.17
-  - Kubernetes v1.18
+Supported container platforms (and architectures):
+  - OpenShift v4.3 (IBM Z and IBM PowerPC)
+  - OpenShift v4.4 (x86 and IBM Z)
+  - OpenShift v4.5 (x86)
+  - Kubernetes v1.17 (x86)
+  - Kubernetes v1.18 (x86)
 
 Supported IBM storage systems:
   - IBM Spectrum Virtualize Family (including IBM Flash family members built with IBM Spectrum Virtualize (FlashSystem 5010, 5030, 5100, 7200, 9100, 9200, 9200R) and IBM SAN Volume Controller (SVC) models SV2, SA2)
   - IBM FlashSystem A9000/R
   - IBM DS8880
   - IBM DS8900
 
-Supported operating systems:
-  - RHEL 7.x (x86 architecture)
-  - RHCOS (x86, IBM Z, and IBM PowerPC architectures)
+Supported operating systems (and architectures):
+  - RHEL 7.x (x86)
+  - RHCOS (x86, IBM Z, and IBM PowerPC)
 
-Full documentation can be found on the [IBM Knowledge Center] (https://www.ibm.com/support/knowledgecenter/SSRQ8T).
+Full documentation can be found on the [IBM Knowledge Center](https://www.ibm.com/support/knowledgecenter/SSRQ8T).
```

Comment:
I'm curious why you removed the space here?

Reply:
before:
![image](https://user-images.githubusercontent.com/57443811/93857332-1804a000-fcc3-11ea-80fb-7063b30ed98a.png)

after:
![image](https://user-images.githubusercontent.com/57443811/93857290-091ded80-fcc3-11ea-84f8-de849ed5e57e.png)